### PR TITLE
ck/COPS-4196 ReRestore backup/restore information for 1.10 and 1.11

### PR DESCRIPTION
**Pull request by ck4adventure**
## Description
https://jira.mesosphere.com/browse/COPS-4196
Restored the older information for 1.10 and 1.11 regarding backup and restoring the IAM database

## Urgency
- [ X] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.10/installing/installation-faq/index.md [hunk 1]
```diff
@@ -50,20 +50,48 @@ If Auto Scaling groups are in use, the node will be replaced automatically.
...
 ## Q. How do I backup the IAM database?
+[/enterprise]
 
-- To backup the IAM database to a file run the following command on one of the master nodes:
+To backup the IAM database to a file run the following command on one of the master nodes:
 
-```bash
-dcos-shell iam-database-backup > ~/iam-backup.sql
-```
+    ```bash
```
Comment (some-things) [inline pages/1.10/installing/installation-faq/index.md]: It looks like the formatting is broken here.

File: pages/1.10/installing/installation-faq/index.md [hunk 2]
```diff
@@ -50,20 +50,48 @@ If Auto Scaling groups are in use, the node will be replaced automatically.
     ```bash
     sudo systemctl kill -s SIGUSR1 dcos-mesos-slave-public && sudo systemctl stop dcos-mesos-slave-public
     ```
+
+[enterprise]
 ## Q. How do I backup the IAM database?
+[/enterprise]
 
-- To backup the IAM database to a file run the following command on one of the master nodes:
+To backup the IAM database to a file run the following command on one of the master nodes:
```
Comment (some-things) [inline pages/1.10/installing/installation-faq/index.md]: Add comma:
> To backup the IAM database to a file, run the following command on one of the master nodes:

Comment (ck4adventure) [reply pages/1.10/installing/installation-faq/index.md]: Added commas

File: pages/1.10/installing/installation-faq/index.md (hunk 1 above)
Comment (ck4adventure) [reply pages/1.10/installing/installation-faq/index.md]: That was intentional. The scripts that are used to convert the markdown to html actually need those indents to format the code block properly.

**Pull request merged**